core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -240,7 +248,15 @@ public T removeAcl(List<AclEntry> entries) throws IOException {
    * @return the updated object
    */
   public T replaceAcl(List<AclEntry> entries) {
-    mAcl.clearEntries();
+    boolean clearACL = false;
+    for (AclEntry entry : entries) {
+      if (!entry.isDefault()) {
```

Comment:
Please add comment why a default entry matters to clearing or not.

Reply:
added